client/js/components/tags/TagListEditForm.vue

```diff
@@ -128,18 +133,26 @@ export default {
       this.$emit('abort')
     },
 
-    deleteItem () {
-      const { type, attributes, children, id } = this.nodeElement
+    async deleteItem () {
+      const { type, id } = this.nodeElement
+
+      if (this.$refs.confirmDialog?.open) {
+        const isConfirmed = await this.$refs.confirmDialog.open()
+
+        if (isConfirmed) {
+          this.$emit('delete', { id, type })
```

Comment:
Why did you change the event payload?

Reply:
what do you mean by 'changed the payload'? Here --> `this.$emit('delete', { id, type })`? It was so before

Reply:
ah i see, you're right. only the destructured nodeElement props are reduced to the ones you actually use.